mlrun/runtimes/remotesparkjob.py

```diff
@@ -129,6 +131,12 @@ def spec(self, spec):
 
     def with_spark_service(self, spark_service, provider=RemoteSparkProviders.iguazio):
         """Attach spark service to function"""
+        # check for existence of spark_services
```

Comment:
@guy1992l @davesh0812 the test should not be in the client (w/o k8s access), rather in the server side when this value is used for deployment (and there is k8s)